instances/latest/commonCoordinateSpaceVersions/MarmosetNMT/MarmosetNMT_v1.jsonld

```diff
@@ -0,0 +1,117 @@
+    "accessibility": {
+        "@id": "https://openminds.om-i.org/instances/productAccessibility/freeAccess"
+    },
+    "anatomicalAxesOrientation": {
+        "@id": "https://openminds.om-i.org/instances/anatomicalAxesOrientation/RAS"
+    },
+    "versionIdentifier": "v1",
+    "fullDocumentation": {
+        "@type": "https://openminds.om-i.org/types/WebResource",
+        "contentDescription": null,
+        "format": "https://openminds.om-i.org/instances/contentTypes/text_html",
+        "IRI": "https://www.marmosetbrain.org/nencki_monash_template/"
+    },
+    "nativeUnit": {
+        "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
+    },
+    "axesOrigin": [
+        {
+            "@type": "https://openminds.om-i.org/types/QuantitativeValue",
+            "typeOfUncertainty": null,
+            "uncertainty": null,
+            "unit": {
+                "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
+            },
+            "value": 0
+        },
+        {
+            "@type": "https://openminds.om-i.org/types/QuantitativeValue",
+            "typeOfUncertainty": null,
+            "uncertainty": null,
+            "unit": {
+                "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
+            },
+            "value": 0
+        },
+        {
+            "@type": "https://openminds.om-i.org/types/QuantitativeValue",
+            "typeOfUncertainty": null,
+            "uncertainty": null,
+            "unit": {
+                "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
+            },
+            "value": 0
+        }
+    ],
+    "releaseDate": "2021-02-01",
+    "copyright": {
+        "@type": "https://openminds.om-i.org/types/Copyright",
+        "holder": {
+            "@type": "https://openminds.om-i.org/types/Organization",
+            "fullName": "Elsevier Inc."
+        },
+        "year": "2021"
+    },
+    "usedSpecimen": [
+        {
+            "@type": "https://openminds.om-i.org/types/SubjectGroup",
+            "species": {
+                "@id": "https://openminds.om-i.org/instances/species/callithrixJacchus"
+            },
+            "studiedState": [
+                {
+                    "@type": "https://openminds.om-i.org/types/SubjectGroupState",
+                    "ageCategory": [
+                        {
+                            "@id": "https://openminds.om-i.org/instances/ageCategory/youngAdult"
+                        }
+                    ]
+                }
+            ],
+            "numberOfSubjects": 20,
+            "biologicalSex": [
+                {
+                    "@id": "https://openminds.om-i.org/instances/biologicalSex/male"
+                },
+                {
+                    "@id": "https://openminds.om-i.org/instances/biologicalSex/female"
+                }
+            ]
+        }
+    ],
```

Comment:
this cannot be nested but has to be linked objects. For now please delete this part in full (set usedSpecimen to null) because we do not want to create duplicates in the adopting systems (e.g. EBRAINS) which register the subjects with the datsets. We can add them later again if we know how to solve this issue across databases.